src/main/java/gov/nih/nci/evs/api/service/SparqlQueryManagerServiceImpl.java

```diff
@@ -1183,28 +1183,32 @@ private Concept getConceptByType(final String conceptType, String conceptCode,
   @Override
   public void checkPathInHierarchy(String code, HierarchyNode node, Path path,
     Terminology terminology) throws JsonParseException, JsonMappingException, IOException {
-    if (path.getConcepts().size() == 0) {
+    if (path.getConcepts().size() == 0) { // check for empty path
```

Comment:
Hey, this is mostly a style thing, but there are issues (especially when formatting code) to putting comments at the end of the line - and actually some code style checks will fail this as a violation.   Comments are super important and we need them here to explain what's going on, but do me the favor of just moving them up before the line, as in...

// check for empty path
if (path.getConcepts().size() == 0) {
...

Reply:
should i do that for all the comments in that function or just the one you pointed out?